.githooks/pre-commit

```diff
@@ -77,7 +76,7 @@ staged_swift_files.each { |filename|
 print "Linting Swift in your changed Swift files (if any)...\n"
 result = `#{compose_swiftlint_command(relative_staged_swift_files, false)}`
 unless result.empty? 
-    abort(result)
+    print result
```

Comment:
is it from develop? or e2ei is not completly up to date?

Reply:
I don't see this change in the diff

Reply:
I don't know what happened 😅